faucet-server/Sources/App/Controllers/CKBController.swift

```diff
@@ -80,11 +80,15 @@ public struct CKBController: RouteCollection {
     // MARK: - Utils
 
     public func sendCapacity(address: String) throws -> H256 {
+        let nodeUrl = URL(string: Environment.Process.nodeURL)!
         let api = APIClient(url: URL(string: Environment.Process.nodeURL)!)
+        let systemScript = try SystemScript.loadFromGenesisBlock(nodeUrl: nodeUrl)
```

Comment:
TODO: Consider cache `systemScript`'s code hash and out point data in the future so it doesn't have to load the genesis block every time we send out some capacity.

Reply:
@ashchan  Ok, I have already cached `SystemScript`. 
https://github.com/nervosnetwork/ckb-testnet-faucet/pull/67/commits/c087eb469241b8fd2b2b3ae6ebb08c6d302f7bed

Reply:
nice